packages/js/src/ui/config/defaultLocalization.ts

```diff
@@ -1,17 +1,20 @@
+// TODO check all localization keys
 export const defaultLocalization = {
   locale: 'en-US',
-  'inbox.title': 'Inbox',
```

Comment:
not used

Reply:
This was probably the first one we added as an example!